samples/VM-Right-Sizing/wf_host_create_ticket.json

```diff
@@ -0,0 +1,133 @@
+        "attachmentToggleValue": "none"
+      },
+      "position": {
+        "x": 0,
+        "y": 4
+      },
+      "predecessors": [
+        "move_to_pending"
+      ],
+      "conditions": {
+        "states": {
+          "move_to_pending": "OK"
+        }
+      },
+      "retry": {
+        "count": 2,
+        "delay": 10,
+        "failedLoopIterationsOnly": true
+      },
+      "concurrency": 1,
+      "withItems": "issue in {% if result(\"get_cpu\").records | length > 1 %} {{result(\"create_issue\")}} {% else %} [{{result(\"create_issue\")}}] {% endif %}"
+    },
+    "create_issue": {
+      "name": "create_issue",
+      "description": "Open a ticket in JIRA for each host.",
+      "action": "dynatrace.jira:jira-create-issue",
+      "input": {
+        "labels": [],
+        "project": {
+          "id": "10001"
+        },
+        "summary": "Hosts with possible upsize/downsize",
+        "assignee": null,
+        "priority": null,
+        "reporter": {
+          "id": "63c15bf62341bff4fff70812"
+        },
+        "issueType": {
+          "id": "10011"
+        },
+        "components": [],
+        "description": "{{ _.host.Host }} has a CPU of {{ _.host.CPU }}",
+        "connectionId": "vu9U3hXa3q0AAAABAB1hcHA6ZHluYXRyYWNlLmppcmE6Y29ubmVjdGlvbgAGdGVuYW50AAZ0ZW5hbnQAJGJhY2IzNzBlLTI4ZjctMzE3YS04OTZjLTAzMWIwMDc4YmE4Yb7vVN4V2t6t",
+        "fieldSetters": []
+      },
+      "position": {
+        "x": 0,
+        "y": 2
+      },
+      "predecessors": [
+        "get_cpu"
+      ],
+      "conditions": {
+        "states": {
+          "get_cpu": "OK"
+        }
+      },
+      "concurrency": 1,
+      "withItems": "host in {{ result('get_cpu').records }}"
+    },
+    "move_to_pending": {
+      "name": "move_to_pending",
+      "description": "Transition the issue to 'pending'",
+      "action": "dynatrace.jira:jira-transition-issue",
+      "input": {
+        "issue": "{{ _.issue.id }}",
+        "comment": "DT Workflow moving to pending",
+        "project": "10001",
+        "issueType": "10011",
+        "connectionId": "vu9U3hXa3q0AAAABAB1hcHA6ZHluYXRyYWNlLmppcmE6Y29ubmVjdGlvbgAGdGVuYW50AAZ0ZW5hbnQAJGJhY2IzNzBlLTI4ZjctMzE3YS04OTZjLTAzMWIwMDc4YmE4Yb7vVN4V2t6t",
+        "fieldSetters": [],
+        "targetStatus": "10009"
+      },
+      "position": {
+        "x": 0,
+        "y": 3
+      },
+      "predecessors": [
+        "create_issue"
+      ],
+      "conditions": {
+        "states": {
+          "create_issue": "OK"
+        },
+        "custom": ""
+      },
+      "retry": {
+        "count": 2,
+        "delay": 5,
+        "failedLoopIterationsOnly": true
+      },
+      "concurrency": 1,
+      "withItems": "issue in {% if result(\"get_cpu\").records | length > 1 %} {{result(\"create_issue\")}} {% else %} [{{result(\"create_issue\")}}] {% endif %}"
+    }
+  },
+  "actor": "0bc2624c-f423-44e1-b766-02c0a64ca725",
+  "owner": "3c14a43c-e9c2-4ce6-817f-737cb977189a",
```

Comment:
at least owner ids must be removed so that it can be imported in another account (a limitation that hopefully should be removed and this id should just be ignored)

Reply:
Removed this.